src/client/chessboard/chessboard-wrapper.tsx

```diff
@@ -119,7 +112,6 @@ export function ChessboardWrapper(props: ChessboardWrapperProps): JSX.Element {
 
                     const isSquareLegalMove =
                         lastClickedSquare !== undefined &&
```

Comment:
This line is redundant, which might make this variable declaration also redundant.